test/tests/kernels/simple_transient_diffusion/simple_transient_diffusion.i

```diff
@@ -1,3 +1,13 @@
+[Outputs]
+  [png]
+    type = PNGObject
+    PNGFile = "PNGTEST"
```

Comment:
Let's not update this test. Create a new test (or two or three) in the test/tests/outputs/png/ folder (or something similar)